src/pages/settings/personal.svelte

```diff
@@ -0,0 +1,169 @@
+      setNotice('Notification option has been saved')
+    }
+
+    const handleUnchecked = (policyId: string) => {
+      setNotice('Notification option has been saved')
+    }
+
+    async function onFileSelect(event: CustomEvent<FormData>) {
+      croppie = croppie || new Croppie(document.getElementById('croppie-container'), {
+        viewport: { width: 128, height: 128, type: 'circle' }
+      })
+
+      let file = event.detail.get('file') as File
+
+      if (file) {
+        let reader = new FileReader()
+        reader.onload = e => {
+          croppie.bind({
+            url: e.target.result as string
+          })
+        }
+
+        reader.readAsDataURL(file)
+      }
+    }
+
+    async function onUpload() {
+      if (!croppie) {
+        setNotice('Please select an image to upload')
+      } else {
+        try {
+            uploading = true
+
+            croppie.result({ type: 'blob', circle: true }).then(async result => {
+              let data = new FormData()
+              data.append('file', result)
+
+              const file = await upload(data)
+
+              updateUser({
+                ...$user,
+                photo_file: {
+                  content_type: file.content_type,
+                  created_by_id: $user.id,
+                  id: file.id,
+                  name: file.filename,
+                  size: file.size,
+                  url: file.url,
+                  url_expiration: undefined
+                },
+                photo_file_id: file.id
+              })
+
+              setNotice('Your profile photo has been updated')
+            })
+          } catch {
+            setNotice('There was an error uploading your profile photo')
+          } finally {
+              uploading = false
+        }
+      }
+    }
+    
+    const isEmailValid = (email: string) => email.includes('@') // bare basic validation
+    const isLocationValid = (location: string) => !!location
+</script>
+    
+<style>
+.required {
+  color: var(--mdc-theme-status-error);
+}
+
+.photo-preview {
+  width: 300px;
+  height: 300px;
+  padding-bottom: 40px;
+}
+</style>
+    
+<Page>
+  <Breadcrumb />
+
+  <h3 class="ml-1 mt-3">Notification email</h3>
+  <p>
+      <RadioOptions name="notificationEmail" options={notificationOptions} bind:value={notification_email} ></RadioOptions>
+  </p>
+  {#if notification_email === NOTIFICATION_OPTION_CUSTOM }
+  <TextField placeholder={'Custom email'} bind:value={email_override} on:blur={updateCustomEmail} ></TextField>
+  {/if}
+
+  <h3 class="ml-1 mt-3">Location<span class="required">*</span></h3>
+  <p>
+    <TextField placeholder={'Enter country'} bind:value={location} on:blur={updateLocation} />
+  </p>
+
+  <h3 class="ml-1 mt-3">Receive notification emails for</h3>
+  <p>
+      {#each $policies as policy (policy.id) }
+      <Checkbox label={policy.type} checked={true} on:checked={() => handleChecked(policy.id)} on:unchecked={() => handleUnchecked(policy.id)} ></Checkbox>
```

Comment:
Another simplification Svelte allows is that you can leave off the value of an attribute if you simply want it to be `true`, which can make things like `checked` shorter yet still clearly understandable.
```suggestion
      <Checkbox label={policy.type} checked on:checked={() => handleChecked(policy.id)} on:unchecked={() => handleUnchecked(policy.id)} ></Checkbox>
```